docs/en/observability/uptime-set-up-choose-project-monitors.asciidoc

```diff
@@ -42,41 +63,61 @@ You should also decide where you want to run the monitors before getting started
 * *Private locations*: Private locations allow you to run monitors from your own premises.
   To use private locations will must create a private location before continuing.
   For step-by-step instructions, refer to <<synthetics-private-location>>.
-// end::prereq[]
+====
+
 
 [discrete]
-[[uptime-set-up-project-monitors-add]]
-== Add monitors
-
-To use a YAML file and the Elastic Synthetics library to add a
-lightweight monitor:
-
-. You'll need a project using `@elastic/synthetics`. You can use an existing project where you manage browser monitors, or you can create a new project using `elastic-synthetics init`.
-. Add one or more `.yml` files to the project.
-. Configure monitors in the `.yml` files. For example:
-
-```yml
-heartbeat.monitors:
-- type: http
-  id: elastic-http
-  name: Elastic HTTP check
-  urls: ["https://elastic.co"]
-  schedule: "@every 60s"
-  tags: ["org:elastic"]
-- type: tcp
-  id: elastic-tcp
-  name: Elastic TCP check
-  hosts: ["elastic.co"]
-  schedule: "@every 60s"
-  tags: ["org:elastic"]
-```
+== Create a project
 
-[IMPORTANT]
-====
-*This functionality is still in beta and not all {heartbeat} configuration options are supported.* If you attempt to push a project monitor with a configuration option that is not supported, the push will fail with an error.
-====
+Start by creating your first project. Run the command below to create a new 
+project named `projects-test` in the current directory.
+
+[source,sh]
+----
+npx @elastic/synthetics init projects-test
+----
+
+Then, follow the prompts on screen to setup the correct default variables for your project.
+When complete, set the `SYNTHETICS_API_KEY` environment variable in your terminal, which is used
+for authentication with your Elastic Stack. You will most likely want to set this permanently. 
+This is done differently for https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_environment_variables?view=powershell-7.2#saving-changes-to-environment-variables[Powershell] and https://unix.stackexchange.com/a/117470[Bash].
+
+Then, take a look at key files and directories inside your project:
+
+* `journeys` is where you'll add `.yaml`, `.ts`, and `.js` files defining your monitors. When you create a new project, this directory will contain files defining sample monitors.
+* `synthetics.config.ts` contains settings for your project. When you create a new project, it will contain some basic configuration options that you can customize later.
+* `package.json` contains NPM settings for your project. Learn more in the https://docs.npmjs.com/about-packages-and-modules[Synthetic tests configuration].
+* `.github` contains sample Github actions workflow files for use with Github Actions.
+
```

Comment:
```suggestion

It'd be helpful to just show the files a bit here, people will mostly get what's going on here, and it will help frame things for later sections. Note that I had to break the closing triple ticks for the nested code sections to get formatting here right, you'll need to manually edit this beyond just having git include it.

[discrete]
== Examine the journey files

Inside the `journeys` directory you'll find examples of both Browser and Lightweight monitors. 

Lightweight monitors look like:

```yml
# lightweight.yml
- type: http
  name: Todos Lightweight
  id: todos-lightweight
  urls: "https://elastic.github.io/synthetics-demo/"
  schedule: '@every 1m
#```

While browser monitors look like:

```ts
//example.journey.ts
import { journey, step, monitor, expect } from '@elastic/synthetics';

journey('My Example Journey', ({ page, params }) => {
  // Only relevant for the push command to create
  // monitors in Kibana
  monitor.use({
    id: 'example-monitor',
    schedule: 10,
  });
  step('launch application', async () => {
    await page.goto(params.url);
  });

  step('assert title', async () => {
    const header = await page.locator('h1');
    expect(await header.textContent()).toBe('todos');
  });
});
#```

```